package.json

```diff
@@ -49,15 +49,15 @@
     "heap": "~0.2.6",
     "interface-datastore": "~0.7.0",
     "k-bucket": "^5.0.0",
-    "libp2p-crypto": "~0.16.1",
+    "libp2p-crypto": "~0.17.0",
     "libp2p-record": "~0.6.2",
     "merge-options": "^1.0.1",
-    "multihashes": "~0.4.14",
-    "multihashing-async": "~0.5.2",
+    "multihashes": "~0.4.15",
+    "multihashing-async": "~0.6.0",
     "p-queue": "^5.0.0",
     "p-times": "^2.1.0",
-    "peer-id": "~0.12.2",
-    "peer-info": "~0.15.1",
+    "peer-id": "~0.12.3",
+    "peer-info": "~0.16.0",
```

Comment:
Several of the minor version bumps here are from callbacks to async/await, which are breaking changes. I think most of these should be promisified calls, they will need to be updated to no longer use promisify and use await calls instead.

Reply:
It works also with `peer-info 0.15.1` which comes without promisifications